components/BottomAppBar/README.md

```diff
@@ -106,73 +106,6 @@ import MaterialComponents.MaterialBottomAppBar
 MDCBottomAppBarView can be added to a view hierarchy like any UIView. Material Design guidelines
 recommend always placing the bottom app bar at the bottom of the screen.
 
-
-## Extensions
-
-<!-- Extracted from docs/color-theming.md -->
```

Comment:
We should probably replace themers doc with documentation about theming.
Removing without a replacement might make it seems to some that theming as a whole is no longer supported...

Reply:
Done.